frontend/assets/cards/A3.json

```diff
@@ -5391,7 +5391,7 @@
     "retreat": "N/A",
     "rarity": "◊◊",
     "fullart": "No",
-    "ex": "no",
+    "ex": "yes",
```

Comment:
Why did you change these? These are not ex cards and the rarities should be on the shinies. Did you overwrite by running the scraper?